### DISCUSSION TITLE
🐞 Using old number got "is not on whatsapp"

### DISCUSSION THREAD

**Discussion by aldoexpert** [aldinokemal/go-whatsapp-web-multidevice#172] [Announcements]
### What happened?

saya mengirim ke nomor whatsapp yang memang jumlah digitnya sedikit (nomor lama) dengan jumlah nomor 10 digit dan sudah menggunakan whatsapp sejak lama,
tetapi menggunakan project api ini mengalami kasus dimana gagal mengirim ke nomor yg jumlah digit angka hanya 10 saja karena nomor tidak di whatsapp
`{
    "code": "INVALID_JID",
    "message": "Phone 62814567890@s.whatsapp.net is not on whatsapp"
}`
note: nomor disamarkan
sedangkan jika menggunakan nomor yang jumlah digit 11 - 12 angka bisa berhasil

Salam
Terimakasih

### Reproduction Steps

```bash
1. using api /send/message

2. raw json
{
  "phone": "62814567890@s.whatsapp.net",
  "message": "maaf hanya tes kirim saja\n - by Aldo"
}
```

### Relevant log output

```bash
{
    "code": "INVALID_JID",
    "message": "Phone 62814567890@s.whatsapp.net is not on whatsapp"
}
```

### Application Version

4.14.1

### Verification

- [X] I have checked for existing closed issues and discussions.

**Comment by aldinokemal** [comment]
> karena nomor tidak di whatsapp
> { "code": "INVALID_JID", "message": "Phone 62814567890@s.whatsapp.net is not on whatsapp" }
> note: nomor disamarkan

sudah dicoba dicek disini?
https://wa.me/62814567890

**Reply by aldoexpert** [reply]
https://wa.me/62814567890
note: nomor disamarkan

menggunakan wa.me bisa langsung chat ke nomor yg dituju (nomor yg hanya 10 digit) dan bisa chat wa dgn orang tsb.

**Comment by aldoexpert** [comment]
https://wa.me/62814567890
note: nomor disamarkan

menggunakan wa.me bisa langsung chat ke nomor yg dituju (nomor yg hanya 10 digit) dan bisa chat wa dgn orang tsb.
Tetapi saat menggunakan api dsini tetap mengalami **INVALID_JID**

**Reply by aldinokemal** [reply]
Halo Aldo, 

Kebetulan saya tidak memiliki nomer dengan 10 digit, jadi sepertinya sulit untuk melakukan test khusus seperti ini kecuali kalau kita bisa testing bersama dan anda provide nomor 10 digit

**Reply by aldoexpert** [reply]
saya sudah mengirim melalui email di tanggal 2 agustus, mungkin belum terbaca, di dalam email jg sudah ada nomor yg 10 digit tsb

**Comment by aldinokemal** [comment]
Fixed `v4.17.0`
tambahkan flag untuk menghindari pengecekan
`--account-validation=false`
